apps/docs/content/components/CTABanner/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: CTA banner
+description: Use CTA banner to draw attention to or create urgency around a user action.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing the CTA banner anatomy with a heading, a description and a button group.](https://github.com/primer/brand/assets/912236/779c909d-21eb-42b8-b3dd-963e0113102c)
+
+- Heading: The main title of the Pillar. It should be short and concise.
+- Description: Short text that extends the information provided by the heading.
+- Button group: The main and secondary actions the user should take on the page
```

Comment:
```suggestion
- Button group: The primary and secondary actions available to users
```

Reply:
reading main and secondary felt a bit unnatural, but feel free to keep it if that was intended